### PR TITLE
Filled out rest of decks.asm. Name changes of pokemon to be more descriptive …

### PR DESCRIPTION
**Pull request by re-cache**
…than "MEW1, MEW2," etc., by adding levels to their names as per the game (ex. "MEW_LV15") (a few pokemon levels in GB2 are different than real cards) and these changes are in card_constants.asm as well as decks.asm

Reflected these name changes also in card_collection.asm
Some of the deck names in decks.asm can be changed in the future to be more accurate.

### CODE REVIEW

File: src/constants/card_constants.asm [hunk 1]
```diff
@@ -444,4 +444,4 @@
 	const ROCKETS_SNEAK_ATTACK    ; $1bb
 	const HERE_COMES_TEAM_ROCKET  ; $1bc
 	const THE_ROCKETS_TRAP        ; $1bd
-DEF NUM_CARDS EQU const_value - 1
+DEF NUM_CARDS EQU const_value - 1
```
Comment (dannye) [inline src/constants/card_constants.asm:447]: Can you add back this EOF newline?

File: src/data/decks.asm [hunk 2]
```diff
@@ -163,21 +163,2770 @@ StrangePsyshockDeck: ; 595d2 (16:55d2)
...
+	dbw  1, MEOWTH_LV17
 	dbw  1, ENERGY_RETRIEVAL
 	dbw  2, POTION
 	dbw  1, GUST_OF_WIND
 	dbw  2, SWITCH
 	dbw  1, BILL
 	db 0
-; 0x59637
+
+SweatAntiGR1Deck: ;
```
Comment (dannye) [inline src/data/decks.asm:184]: You can remove all of these empty ` ; ` comments.

File: src/data/decks.asm [hunk 3]
```diff
@@ -163,21 +163,2770 @@ StrangePsyshockDeck: ; 595d2 (16:55d2)
...
+	dbw  1, RATICATE
+	dbw  1, MEOWTH_LV14
+	dbw  1, PROFESSOR_OAK
+	dbw  2, BILL
+	dbw  1, SWITCH
+	dbw  1, COMPUTER_SEARCH
+	dbw  1, PLUSPOWER
+	dbw  2, POTION
+	dbw  2, FULL_HEAL
+	db 0
```
Comment (dannye) [inline src/data/decks.asm:2932]: Missing EOF newline.

File: src/constants/card_constants.asm [hunk 4]
```diff
@@ -10,384 +10,384 @@
...
+	const BULBASAUR_LV15          ; $00e
+	const IVYSAUR_LV20            ; $00f
+	const IVYSAUR_LV26            ; $010
 	const DARK_IVYSAUR            ; $011
-	const VENUSAUR1               ; $012
-	const VENUSAUR2               ; $013
-	const VENUSAUR3               ; $014
+	const VENUSAUR_LV64           ; $012
+	const VENUSAUR_LV67           ; $013
+	const VENUSAUR_LV67A          ; $014
```
Comment (dannye) [inline src/constants/card_constants.asm:21]: I'm warming up to the idea of adding level to the constants but there may be a better suffix to distinguish pairs like these.

If we decide to adopt this change, then the card constants in poketcg 1 should be updated to match. Would you be able to open a similar PR to that repo to keep them in sync? (if not, one of us would get around to it)

**Review by dannye — COMMENTED** [review]
Thanks for this! It's great to see a chunk of progress like this.

File: src/constants/card_constants.asm (hunk 1 above)
Comment (re-cache) [reply src/constants/card_constants.asm:447]: Yes

File: src/data/decks.asm (hunk 3 above)
Comment (re-cache) [reply src/data/decks.asm:2932]: I'll put one in

File: src/data/decks.asm (hunk 2 above)
Comment (re-cache) [reply src/data/decks.asm]: I added deck addresses here to my recent pr update

File: src/constants/card_constants.asm (hunk 4 above)
Comment (re-cache) [reply src/constants/card_constants.asm:21]: I have a list of the 62 names with numbers at the end, how would you like to handle referencing them?

**Pull request merged**